nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java

```diff
@@ -112,11 +112,7 @@ public BrukerData(String path, File nusFile) throws IOException {
         File file = new File(path);
         openParFile(file);
         openDataFile(path);
-        if (dim < 2) {
-            scale = 1.0e6;
-        } else {
-            scale = 1.0e6;
-        }
+        scale = 1.0e6;
```

Comment:
removed the if/else since seems to be in the file from the beginning but double check if scale is actually supposed to be different?

Reply:
This constructor is called only for FID as far as I can see. The other constructor sets scale to `1` and is called for processed data. @brucejohnson can you confirm?

Reply:
This change can be left in, but after this PR is merged I'm going to do some work on setting scale.  There is a Bruker parameter that indicates how the data was scaled during collection and we are ignoring that parameter.  That results in relative intensities being wrong when multiple datasets are collected on same sample (recently pointed out by user).